core/src/main/scala/com.snowplowanalytics.stream/loader/JsonRecord.scala

```diff
@@ -18,10 +18,12 @@
  */
 package com.snowplowanalytics.stream.loader
 
+import org.json4s.JObject
+
 /**
  * Format in which Snowplow events are buffered
  *
  * @param json The JSON string for the event
- * @param id The event ID (if it exists)
+ * @param partition partition to send the data to, currently not used.
  */
-case class JsonRecord(json: String, id: Option[String])
+case class JsonRecord(json: JObject, partition: String)
```

Comment:
can we move this out of this PR, this doesn't seem related to abstracting the core module

Reply:
removed it to have it with partition pr